src/controller/users.js

```diff
@@ -0,0 +1,17 @@
+const db = require("../database/models/index")
+const User = db.User
+let newUser;
+// creating an User 
+exports.createUser = async(req, res) => {
+    let { email, password, confirm_password } = req.body
+    if (!email) return res.status(400).send({ error: "Please provide a email" })
```

Comment:
Use email pattern or use joi bcz if the user type anything here system will let him in which is not good